.azure-pipelines/cpu-unit-test.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+trigger:
+  - main
+
+container:
+  image: python:3.7
+
+steps:
+  - script: |
+      python3 -m pip install .[test,torch]
+    displayName: Install dependencies
+  - script: |
+      python3 setup.py lint
+    displayName: Run code lint
+  - script: |
+      python3 setup.py test
+    displayName: Run unit tests
+    timeoutInMinutes: 10
```

Comment:
timeout will triger failure?

Reply:
I think it will trigger that this test will not pass.

Reply:
yes, it will fail if timeout. this is used to prevent hanging during tests.